sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -455,6 +455,9 @@ object FunctionRegistry {
 
     // aggregate functions
     expression[HyperLogLogPlusPlus]("approx_count_distinct"),
+    expression[HyperLogLogPlusPlusSketch]("approx_count_distinct_sketch"),
```

Comment:
I would recommend to give these shorter names for easier usability in SQL. For example, in prior systems, I've used: HLL_COUNT.INIT, HLL_COUNT.MERGE, and HLLCOUNT.FINALIZE. What do you think?

Reply:
My only worry with this is that by diverging from the 'HyperLogLogPlusPlus' and 'approx_count_distinct' prefixes, the new functions wouldn't be visible in alphabetical order alongside the original. I would prefer the function names to be shorter, but I felt that using the same prefix was a higher priority. Would you mind if we have a larger discussion about naming in the main PR and get opinions from a wider group?